src/app/find-help/material-aid-search/material-aid-search-form/material-aid-search-form.component.html

```diff
@@ -0,0 +1,31 @@
+<form (ngSubmit)="search.emit(data)" #form="ngForm">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4">
+        <label
+          ><b>{{ "TYPE_OF_SUPPORT" | translate }}</b></label
+        >
+        <mat-form-field appearance="outline">
+          <mat-select placeholder="Wybierz z listy" [(ngModel)]="data.category" name="kategoriePomocy">
+            <mat-option *ngFor="let cat of categories" [value]="cat.value">{{ cat.value | translate }}</mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+      <div class="col-md-4">
+        <app-cities-search
+          [(ngModel)]="data.location"
+          name="location"
+          [placeholder]="'Np. Rzeszów, podkarpackie'"
```

Comment:
BTW, we should also translate placeholders. This one is repeated in a few places so it's fine to leave it for now but we'll need to get to it.